src/main/java/com/minecolonies/util/AchievementUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.stats.Achievement;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+/**
+ * todo: explain this class
+ */
+public final class AchievementUtils
+{
+
+    /**
+     * Private Constructor to deny instances
+     */
+    private AchievementUtils() {}
+
+    //todo document
+    public static void syncAchievements(@NotNull final Colony colony)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)